.changelog/3853.doc.md

```diff
@@ -0,0 +1,3 @@
+Update [Security] document
+
+[Security]: security.md
```

Comment:
all caps SECURITY in filename

Reply:
Thanks, @pro-wh! I've fixed this in https://github.com/oasisprotocol/oasis-core/pull/3855.